tutorial/hu.bme.mit.gamma.tutorial.finish/model/Controller/Controller.gcd

```diff
@@ -15,36 +15,17 @@ statechart ControllerStatechart [
 	transition from PriorityPrepares to Secondary when timeout PriorityPreparesTimeout6
 	transition from Priority to PriorityPrepares when timeout PriorityTimeout4
 	transition from Operating to Interrupted when PoliceInterrupt.police / raise PriorityPolice.police; raise
-	SecondaryPolice.police
+	SecondaryPolice . police
```

Comment:
All changes made to this file should be discarded (as the syntax with respect to the used elements did not change, right)?
Edit: I see the action syntax changed is they now are placed inside blocks. Nevertheless, some formatting rules need to be defined later.